src/functions_framework/__init__.py

```diff
@@ -35,6 +41,12 @@
 DEFAULT_SIGNATURE_TYPE = "http"
 
 
+class _EventType(enum.Enum):
+    LEGACY = 1
+    CLOUDEVENT_BINARY = 2
+    CLOUDEVENT_TEXT = 3
```

Comment:
As mentioned before, CloudEvent Text is not an event type.

The logic used here is for a CloudEvent Structured event.
This is the spec: https://github.com/cloudevents/spec/blob/master/http-protocol-binding.md#32-structured-content-mode

`CLOUDEVENT_TEXT` should be `CLOUDEVENT_STRUCTURED`.